src/devices/Ft4222/FtCommon.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System.Device.Ft4222
```

Comment:
Please also change all of these namespaces to instead be under Iot.Device.Ft4222 like the rest of the bindings.

Reply:
I agree we should unify the namespaces

Reply:
OK, so I changed everything for Iot.Device.Ft4222